### PR TITLE
Keynotes so far

### PR DESCRIPTION
**Pull request by mwernimont**
@jread-usgs Heres the keynotes so far.

Also a fix for a figure 2 issue, which is figure 1 now, but still figure2.mustache, anyway...

### CODE REVIEW

File: layout/templates/keynote.mustache [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<div class="keynote" id="{{.}}">
```
Comment (jordansread) [inline layout/templates/keynote.mustache]: I would change this to 

```
<div class="keynote" id="{{id}}">
  <div class="keynoteContent">{{content}}</div>
  <div class="dimmer"></div>
</div>
```

and then make the corresponding change to siteText.yaml (see other comment)

File: data/siteText.yaml [hunk 2]
```diff
@@ -149,22 +148,28 @@ ref3: >-
...
+
+whatAreKnownRisks: keynoteDiv3
+
+canWeKickTheHabit: keynoteDiv4
+
+microplasticsInRivers: keynoteDiv5
+
+plasticsInRiver: keynoteDiv6
+
+whatsNext: keynoteDiv7
```
Comment (jordansread) [inline data/siteText.yaml]: if you use my suggestion below, structure these like this:

```
whatsNext: 
  id: keynoteDiv7
  content: faketext
```

File: data/siteText.yaml [hunk 3]
```diff
@@ -27,15 +27,14 @@ envEffects-oysters2: particles per serving of commercially-cultured <strong>oyst
 envEffects-oysters3: 90
 envEffects-oysters4: "particles per serving of commercially-cultured <strong>mussels</strong><sup><a href='#ref2'>2</a></sup>"
 
-envEffects-stream1: 1 particle for every
-envEffects-stream2: 8
-envEffects-stream3: gallon of Great Lakes tributary <strong>water</strong><sup><a href='#ref3'>3</a></sup>
-envEffects-stream4: "On average"
-envEffects-stream5: 1285
-envEffects-stream6: "particles per square foot of <strong>river sediment</strong><sup><a href='#ref4'>4</a></sup>"
+envEffects-stream1: Up to
```
Comment (ldecicco-USGS) [inline data/siteText.yaml:30]: Woah, why are you going back to this text?

Comment (ldecicco-USGS) [inline data/siteText.yaml:30]: It looks like you need to do an upstream pull. The changes I made to the references need to be retained too.

**Comment by mwernimont** [comment]
@ldecicco-USGS So I did a pull and pushed it up to my repo, did that fix the me messing up the siteText? If not what do I gotta do?

Theres also the final keynote code to look at and test @jread-usgs and @ldecicco-USGS

**Comment by ldecicco-USGS** [comment]
I'm not sure why that didn't help. I guess when this PR is merged, I can just fix it back here.

Maybe crop the top (and bottom) of howArePlastics.JPG?

**Comment by mwernimont** [comment]
So This last commit was to get "alt" text to the BG images. You can't actually put an alt tag on a BG image, but W3C says a title is the same as an alt tag. So I think we are covered. 

Sorry about the extra work I made for you :(

**Comment by mwernimont** [comment]
@ldecicco-USGS Why the crop? Kinda don't wanna do that since all the pics are the same dimensions and will probably make the plactics.jpg to stretch and look distorted...

**Comment by ldecicco-USGS** [comment]
Oh, OK, I just thought that upper corner triangle looked funny. But, I assumed they would just come in with 100% width and "auto" height...don't do it if it stretches anything.

**Comment by ldecicco-USGS** [comment]
Awesome

**Pull request merged**